pkgs/applications/networking/flottbot/default.nix

```diff
@@ -0,0 +1,31 @@
+{ lib, buildGoModule, fetchFromGitHub, nixosTests, nix-update-script }:
+buildGoModule rec {
+  pname = "flottbot";
+  version = "0.12.0";
+
+  src = fetchFromGitHub {
+    owner = "target";
+    repo = pname;
+    rev = version;
+    sha256 = "538c2SkyOYmm84yM0svbPeJtSIMHRI6mot8MIP5wBfc=";
+  };
+
+  vendorSha256 = "sha256-6j1nQBlFqd78v/CNV7ivwYZ5CEzGh+An6xcQr4dAkAs=";
+
+  subPackages = [ ];
+
+  doCheck = false; # Tries to do some networking :(
```

Comment:
If I set this to true the test run just fine o.O

```
nix-build -A flottbot
this derivation will be built:
  /nix/store/h5m8q5j99nsnp7lp9xgpqzijvgcspnid-flottbot-0.12.0.drv
building '/nix/store/h5m8q5j99nsnp7lp9xgpqzijvgcspnid-flottbot-0.12.0.drv'...
unpacking sources
unpacking source archive /nix/store/wk3cahkfvxwklcbdham4xf8gnr5jcsc2-source
source root is source
patching sources
configuring
building
Building subPackage ./cmd/flottbot
Building subPackage ./config-example/scripts
Building subPackage ./core
Building subPackage ./handlers
Building subPackage ./models
Building subPackage ./remote
Building subPackage ./remote/cli
Building subPackage ./remote/discord
Building subPackage ./remote/gchat
Building subPackage ./remote/scheduler
Building subPackage ./remote/slack
Building subPackage ./remote/telegram
Building subPackage ./utils
Building subPackage ./version
buildPhase completed in 38 seconds
running tests
ok      github.com/target/flottbot/core 2.035s
ok      github.com/target/flottbot/handlers     1.018s
ok      github.com/target/flottbot/utils        0.010s
ok      github.com/target/flottbot/version      0.003s
checkPhase completed in 33 seconds
installing
post-installation fixup
shrinking RPATHs of ELF executables and libraries in /nix/store/2g1p03mqx4kjr668zj9xhx3jnnmlk750-flottbot-0.12.0
shrinking /nix/store/2g1p03mqx4kjr668zj9xhx3jnnmlk750-flottbot-0.12.0/bin/scripts
patchelf: cannot find section '.dynamic'. The input file is most likely statically linked
shrinking /nix/store/2g1p03mqx4kjr668zj9xhx3jnnmlk750-flottbot-0.12.0/bin/flottbot
checking for references to /build/ in /nix/store/2g1p03mqx4kjr668zj9xhx3jnnmlk750-flottbot-0.12.0...
patchelf: cannot find section '.dynamic'. The input file is most likely statically linked
patching script interpreter paths in /nix/store/2g1p03mqx4kjr668zj9xhx3jnnmlk750-flottbot-0.12.0
stripping (with command strip and flags -S) in  /nix/store/2g1p03mqx4kjr668zj9xhx3jnnmlk750-flottbot-0.12.0/bin
/nix/store/2g1p03mqx4kjr668zj9xhx3jnnmlk750-flottbot-0.12.0
```

which test did you have prolems with?

Reply:
Don't really remember. Maybe they removed/fixed that test case in `0.12.0`. I'll try agian.

Reply:
Sure